server/src/main/java/fr/uga/l3miage/tp1/exo2/models/NetflixUserEntity.java

```diff
@@ -0,0 +1,28 @@
+package fr.uga.l3miage.tp1.exo2.models;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+
+import fr.uga.l3miage.tp1.exo2.enums.sexType;
+
+@Entity
+@Table(name="netflix_user")
+public class NetflixUserEntity {
+    @Id
+    private int id;
+
+    private String uuid;
+    private String lastname;
+    private String firstname;
+
+    @Enumerated(EnumType.ORDINAL)
+    private sexType sex;
+
+    private LocalDate birthDate;
+
+    @OneToOne()
+    @JoinColumn(name = "netflix_account_id")
+    private NetflixAccountEntity netflixAccount;
```

Comment:
Alors ici la relation est unidirectionnel, donc il faut bien utiliser le @JoinColumn de cette manière : 
```suggestion
    @OneToOne()
    @JoinColumn(name = "uuid_user",referencedColumnName = "uuid")
    private NetflixAccountEntity netflixAccount;
```